views/page-settings.php

```diff
@@ -104,27 +104,40 @@
 				<div class="imagify-settings-main-content imagify-pb0<?php echo $hidden_class; ?>">
 					<div class="imagify-settings-section imagify-clear">
 						<div>
-							<h2 class="imagify-options-title"><?php _e( 'Display Options', 'imagify' ); ?></h2>
-
-							<p class="imagify-options-subtitle"><?php _e( 'Show Toolbar Menu', 'imagify' ); ?></p>
-
-							<div class="imagify-col">
-								<p>
+							<h2 class="imagify-options-title"><?php _e( 'Our Plugins', 'imagify' ); ?></h2>
+							<p class="imagify-options-subtitle"><?php _e( 'Build better, faster, safer', 'imagify' ); ?></p>
+							<p class="">
 								<?php
-								$settings->field_checkbox( array(
-									'option_name' => 'admin_bar_menu',
-									'label'       => __( 'I want this awesome quick access menu on my Toolbar.', 'imagify' ),
-								) );
-								?>
-								</p>
-							</div>
-							<div class="imagify-col">
-								<p>
-									<img class="imagify-menu-bar-img" src="<?php echo esc_url( IMAGIFY_ASSETS_IMG_URL . 'imagify-menu-bar-' . $lang . '.jpg' ); ?>" width="273" height="239" alt="">
-								</p>
-							</div>
-
-							<?php
+								_e( 'Beyond Imagify, there\'s a whole family of plugins designed to help you build better, faster and safer websites. Each one is crafted with our unique blend of expertise, simplicity and
+outstanding support. Combine our plugins below to build incredible WordPress webistes!', 'imagify');
+								?>	
+							</p>
+							<?php foreach ( $data['plugin_family'] as $plugin => $plugin_data ) : ?>
+								<div class="imagify-plugin-family-col">
+									<div class="imagify-card">
+										<div class="imagify-card-header">
+											<div class="imagify-card-logo">
+												<img src="<?php echo IMAGIFY_ASSETS_IMG_URL . esc_attr( $plugin_data['logo']['file'] ); ?>" style="width: <?php echo esc_attr( $plugin_data['logo']['width'] ); ?>">
```

Comment:
```suggestion
												<img src="<?php echo esc_url( IMAGIFY_ASSETS_IMG_URL .  $plugin_data['logo']['file'] ); ?>" style="width: <?php echo esc_attr( $plugin_data['logo']['width'] ); ?>">
```

@piotrbak @DahmaniAdame Should we add optimization hints on those images? (like lazyload/decode/fetchpriority)

Reply:
`loading="lazy"` sounds very safe. For others let's wait and do it as a separate project for all the images